glustercli/cmd/glustershd.go

```diff
@@ -82,3 +84,45 @@ var selfHealInfoCmd = &cobra.Command{
 		}
 	},
 }
+
+var selfHealIndexCmd = &cobra.Command{
+	Use:   "index <volname>",
+	Short: "Index Heal",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		var err error
+		volname := args[0]
+		err = client.SelfHeal(volname, "index")
+		if err != nil {
+			if verbose {
+				log.WithFields(log.Fields{
+					"volume": volname,
+					"error":  err.Error(),
+				}).Error("failed to heal the volume")
+			}
+			failure(fmt.Sprintf("Failed to run heal for volume %s\n", volname), err, 1)
+		}
+		fmt.Println("Heal on volume has been successfully launched. Use heal info to check status")
+	},
+}
+
+var selfHealFullCmd = &cobra.Command{
+	Use:   "full <volname>",
+	Short: "Full Heal",
```

Comment:
Short info can be more descriptive. or you can add ```Long : descriptive info about this functionality```

This will be useful if we use cli help command for this